### PR TITLE
{AKS} change the location of kms-related test from centraluseuap to westcentralus

### PR DESCRIPTION
**Pull request by bingosummer**
**Related command**

**Description**

change the location of kms-related test from centraluseuap to westcentralus.
Only for test.

**Testing Guide**

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [x] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py [hunk 1]
```diff
@@ -8440,7 +8440,7 @@ def test_aks_update_with_azurekeyvaultkms_public_key_vault(self, resource_group,
 
     @live_only()
     @AllowLargeResponse()
-    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='centraluseuap', preserve_default_location=True)
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westcentralus', preserve_default_location=True)
```
Comment (FumingZhang) [inline src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:8443]: please also remove `preserve_default_location=True`

Comment (FumingZhang) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py:8443]: what about the following cases?
- test_aks_update_with_azurekeyvaultkms_private_key_vault
- test_aks_create_with_azurekeyvaultkms_private_cluster_v1_private_key_vault

Could you please help update the cases in aks-preview?

Comment (bingosummer) [reply src/azure-cli/azure/cli/command_modules/acs/tests/latest/test_aks_commands.py]: fixed

**Review by FumingZhang — APPROVED** [review]
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=90758462&view=results)

- test_aks_create_with_azurekeyvaultkms_public_key_vault
- test_aks_update_with_azurekeyvaultkms_public_key_vault
- test_aks_create_with_azurekeyvaultkms_private_key_vault
- test_aks_update_with_azurekeyvaultkms_private_key_vault
- test_aks_create_with_azurekeyvaultkms_private_cluster_v1_private_key_vault
- test_aks_disable_azurekeyvaultkms

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**